tools/indexer/example/src/main.rs

```diff
@@ -269,11 +269,13 @@ fn main() {
                 sync_mode: near_indexer::SyncModeEnum::FromInterruption,
                 await_for_node_synced: near_indexer::AwaitForNodeSyncedEnum::WaitForFullSync,
             };
-            actix::System::new().block_on(async move {
+            let runner = actix::System::new();
```

Comment:
Could you please renamed it to `system` we're trying to use it everywhere in this case.

```suggestion
            let system = actix::System::new();
```

Reply:
Done.